processors/tenantidprocessor/tenantidprocessor_test.go

```diff
@@ -173,7 +183,11 @@ func TestReceiveOTLPGRPC_Traces(t *testing.T) {
 		},
 	)
 	require.NoError(t, err)
-	reqTraces := GenerateTraceDataOneSpan()
+
+	err = tracesExporter.Start(context.Background(), componenttest.NewNopHost())
```

Comment:
This was the change that was missing and was causing more pain. Amazingly this is a breaking change that is not explicit in the API but depends on the internal implementation.